taskvine/src/manager/taskvine.h

```diff
@@ -889,6 +889,12 @@ void vine_manager_install_library( struct vine_manager *m, struct vine_task *t,
 */
 void vine_manager_remove_library( struct vine_manager *m, const char *name );
 
+/** Check if a given library name exists on the manager.
+@param m A manager object
+@param library_name The library name to be checked
+*/
+int vine_manager_check_library_exists(struct vine_manager *m, const char *library_name);
```

Comment:
`struct vine_task *vine_manager_get_library(...)` and in python check for `None`.